//
// Created by william on 27/08/18.
//

#ifndef CWRU_DAVINCI_SKILLS_AND_DEMO_DAVINCISKILLSCOMMUNICATIONSACTIONCLIENT_H
#define CWRU_DAVINCI_SKILLS_AND_DEMO_DAVINCISKILLSCOMMUNICATIONSACTIONCLIENT_H

class DavinciSkillsCommunicationsActionClient {

};

#endif //CWRU_DAVINCI_SKILLS_AND_DEMO_DAVINCISKILLSCOMMUNICATIONSACTIONCLIENT_H
